### DISCUSSION TITLE
Possibility to get some jinja templates?

### DISCUSSION THREAD

**Discussion by Headkillah** [jazzyisj/home-assistant-config#60] [General]
Hello jazzyisj.

First, BIG THX for sharing your code, it´s really awesome and i´ve learned much about HA. But, sometimes you use jinja templates like "media_player.jinja" or "style.jinja"

It´s possible to get this files or, if you don´t want to made them public, give some code snippets / hints how you did it from them? I would really appreciate it :)

Thx for your answer.

**Comment by jazzyisj** [comment]
Oops!  I forgot to add the custom_templates folder into my gitignore file!  Thanks for pointing it out :-)

https://github.com/jazzyisj/home-assistant-config/tree/master/custom_templates

**Reply by Headkillah** [reply]
you´re welcome :) Thx for the macros :) Anyway, damn good job keep running.
